app/gui2/stories/GraphEditor.story.vue

```diff
@@ -32,6 +35,20 @@ main =
     result = run_main benches
     third_node = 2 + 2
 `)
+
+/// Note: These props should be synced with the props in
+// `app/ide-desktop/lib/dashboard/src/authentication/src/components/app.tsx`.
+// We need this here, as the rect component is not part of the dashboard and not usually available in the demo scenes.
+// Using this wrapper enables us to see toasts in the absence of the dashboard/React.
+const toastProps = reactive({
+  position: 'top-center',
+  theme: 'light',
+  closeOnClick: false,
+  draggable: false,
+  toastClassName: 'text-sm leading-170 bg-frame-selected rounded-2xl backdrop-blur-3xl',
+  limit: 3,
+})
+const JSONView = createReactWrapper(ToastContainer, toastProps)
```

Comment:
just want to confirm that name of `JSONView` is intentional?

Reply:
I don't think this is at all necessary. The react app already renders toast container, so issued toasts will already be linked to it automatically. We don't need to render additional container.

Reply:
Changed the name to something more sensible.

@Frizi This is our test environment. There is otherwise no react app running there, as we don't initialise the dashboard here, and thus no toast appears. Adding this, ensures the toast container is initialised in the demo scene and allows them to appear. Unless I am missing something?